ldap/servers/slapd/back-ldbm/db-bdb/bdb_layer.c

```diff
@@ -5785,9 +5785,8 @@ bdb_restore(struct ldbminfo *li, char *src_dir, Slapi_Task *task)
 static char *
 bdb__import_file_name(ldbm_instance *inst)
 {
-    char *fname = slapi_ch_smprintf("%s/.import_%s",
-                                    inst->inst_parent_dir_name,
-                                    inst->inst_dir_name);
+    struct ldbminfo *li = inst->inst_li;
+    char *fname = slapi_ch_smprintf("%s/.import_%s", li->li_directory, inst->inst_name);
     slapi_log_err(SLAPI_LOG_DEBUG, "bdb__import_file_name", "DBG: fname=%s\n", fname);
```

Comment:
It's not related exactly to this PR, but it still caught my eye. Do you know of the double underscore (bdb__) is intentional in this function name or just a typo?

Reply:
There is no name conflict so IMHO it is a typo, I will change it.